db/seeds.rb

```diff
@@ -5,29 +5,33 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Organization.destroy_all
+Compliment.destroy_all
+
 messages = [
-  "You rock",
   "You look nice today",
+  "You look dashing today",
+  "You are a genuine person",
+  "You are very wise",
+  "You are an amazing person",
+  "You are very intelligent",
+  "You have good looks and brains",
+  "You are a shrewd decision maker",
+  "You have impeccable timing",
+  "You have a beautiful soul",
+  "You are one of a kind",
 ]
 messages.map { |message| Compliment.find_or_create_by(message: message) }
 
 organizations = [{
   name: 'RailsBridge',
-  description: 'RailsBridge workshops are a free and fun way to get started or level up with web technologies.',
-  reason_to_donate: 'I feel like diversity in tech is important',
+  description: "Railsbridge runs free educational programming workshops targeted at diverse groups of people. They support the idea that the people making technology should accurately reflect the diversity of those using it. Julian is a volunteer instructor for Railsbridge, and Michelle has attended multiple workshops as a student.",
+  reason_to_donate: 'I support inclusive education in tech',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.